editor/jest.config.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
         '\\.[jt]sx?$': 'babel-jest',
       },
       roots: ['src', 'node_modules', '<rootDir>/node_modules'],
-      transformIgnorePatterns: ['/node_modules/(?!utopia-api)'], // this lets ts-jest work on `/node_modules/utopia-api` which is a simlink to `../utopia-api`.
+      transformIgnorePatterns: ['<rootDir>/node_modules/.pnpm/(?!file+..+)'],
```

Comment:
I'm honestly not sure how this hadn't broken before, but it seems like we were on the edge of this breaking since we first switched to pnpm